commons/src/main/kotlin/eu/miaplatform/commons/client/HeadersToProxy.kt

```diff
@@ -1,59 +0,0 @@
-package eu.miaplatform.commons.client
```

Comment:
Why has this file been deleted?

Reply:
It was just moved around / renamed, not deleted. The templates got a pretty big code overhaul while keeping the same functionalities.